src/sentry/auth/idpmigration.py

```diff
@@ -15,8 +44,16 @@ def create_verification_key(user: User, org: Organization, email: str) -> None:
     :param org: the organization whose SSO provider is being used
     :param email: the email address associated with the SSO identity
     """
-    key = str(uuid.uuid4())  # noqa
-    raise NotImplementedError  # TODO
+    cluster = redis.clusters.get("default").get_local_client_for_key(_REDIS_KEY)
+    member_id = OrganizationMember.objects.get(organization=org, user=user).id
+
+    verification_code = get_random_string(32, string.ascii_letters + string.digits)
+    verification_key = f"auth:one-time-key:{verification_code}"
+    verification_value = {"user_id": user.id, "email": email, "member_id": member_id}
```

Comment:
to be extra careful, could we also store in the verification value?:
`identity["id"]` (have to pass this into method)
and the organization_id?

Reply:
Per discussion in meeting, 👍 for `identity["id"]` but org ID can be retrieved from `member_id`.